### PR TITLE
Add PHP quote service

### PR DESCRIPTION
**Pull request by joshleecreates**
This is tested and working as expected, with traces from shipping and quote services. It's a little bit kludgy to accommodate the slight variances in how each service wants the environment variables formatted w/ or w/out protocols, but it all works (even without the ffs port fix).

Fixes https://github.com/open-telemetry/opentelemetry-helm-charts/issues/397

### CODE REVIEW

File: charts/opentelemetry-demo/Chart.yaml [hunk 1]
```diff
@@ -11,4 +11,4 @@ maintainers:
   - name: tylerhelmuth
   - name: wph95
 icon: https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png
-appVersion: "0.4.0-alpha"
+appVersion: "0.5.0-alpha"
```
Comment (TylerHelmuth) [inline charts/opentelemetry-demo/Chart.yaml:14]: Version bump should be done in its own PR, hopefully https://github.com/open-telemetry/opentelemetry-helm-charts/pull/400

**Review by TylerHelmuth — COMMENTED** [review]
@joshleecreates thanks for handling this.  I agree, the env vars are a mess right now, I plan to take care of it soon once things slow down with the demo's feature freeze.  I'd prefer the image bump to be in its own PR but if things can't go independently I understand.  If we do need to bump the version in this PR then the chart's version should bump the minor version instead of the patch.  Also, please handle merge conflict.  @dmitryax please review.

File: charts/opentelemetry-demo/Chart.yaml (hunk 1 above)
Comment (joshleecreates) [reply charts/opentelemetry-demo/Chart.yaml:14]: Will wait for that PR and then bump the chart version.

Comment (TylerHelmuth) [reply charts/opentelemetry-demo/Chart.yaml:14]: @joshleecreates if you tested on 0.5.0-alpha, please comment on that PR to confirm it works as expected.

Comment (joshleecreates) [reply charts/opentelemetry-demo/Chart.yaml:14]: > I plan to take care of it soon once things slow down with the demo's feature freeze. 

Agreed, this makes the most sense.

> I'd prefer the image bump to be in its own PR but if things can't go independently I understand. If we do need to bump the version in this PR then the chart's version should bump the minor version instead of the patch. 

I leave it up to your discretion how you'd like to merge things. I've tested all of the variations and here is the status:

- with #400, (as with the current version of the chart), the shipping service fails because the quote service is not present and the shipping service has no address for it.
- with this PR and app release 0.4.0-alpha, the shipping service error is fixed and the communication between shipping/quote service seems to working, but the quote service fails to send traces to the collector.
- with this PR and app release 0.5.0-alpha, all services work and send traces.

> Also, please handle merge conflict.
✅

Comment (TylerHelmuth) [reply charts/opentelemetry-demo/Chart.yaml:14]: @joshleecreates 0.5.0-alpha change merged in via #400.  Please rebase.

File: charts/opentelemetry-demo/values.yaml [hunk 2]
```diff
@@ -195,4 +211,4 @@ opentelemetry-collector:
         traces:
           exporters:
             - logging
-            - jaeger
+            - jaeger
```
Comment (dmitryax) [inline charts/opentelemetry-demo/values.yaml:214]: Please revert this line

**Review by dmitryax — APPROVED** [review]

**Review by TylerHelmuth — APPROVED** [review]
